src/main/java/com/vestmark/bitbucket/plugin/MergeConflictDetectorService.java

```diff
@@ -0,0 +1,204 @@
+/*
```

Comment:
This is just a slightly modified version of MergeConflictDetectorServlet.java.  It appears as totally new in the diff because of the file name change from MergeConflictDetectorServlet.java to MergeConflictDetectorService.java.

Slight differences:
-The getMergeResults() method replaces the overridden goGet() method in MergeConflictDetectorServlet.java, and handles processing of the new rest service.
-The Repo Id and PullRequest Id are being passed in as rest endpoint parameters instead of being stripped from the Servlet URL.
-The response is a list of JSON objects of class MergeRequestModel which is rendered by merge-conflict-detector-extension.js, instead of a soy template.
The render() method has been removed as it is no longer needed.

The rest of the code is essentially the same as the MergeConflictDetectorServlet.java.